README.md

```diff
@@ -13,6 +13,8 @@ The buildpack will do the following:
 * Contributes `Implementation-Title` manifest entry to `org.opencontainers.image.title` image label
 * Contributes `Implementation-version` manifest entry to `org.opencontainers.image.version` image label
 * Contributes dependency information extracted from Maven naming conventions to the image's BOM
+* Contributes Spring Cloud Bindings as an application dependency (https://github.com/spring-cloud/spring-cloud-bindings).
```

Comment:
This should be `[Spring Cloud Bindings][b]` and the pair of these should be "remote" declarations like `[b]: https://github.com/spring-cloud/spring-cloud-bindings`.